projects/solar/rules.mk

```diff
@@ -6,7 +6,7 @@
 # $(T)_SRC: $(T)_DIR/src{/$(PLATFORM)}/*.{c,s}
 
 # Specify the libraries you want to include
-$(T)_DEPS := ms-common ms-drivers
+$(T)_DEPS := ms-helpers ms-common ms-drivers
```

Comment:
I think this should be `ms-helper`
Looks like GitHub actions is failing because of this as well.

```
T: test_adc_read_runner.c -> test_adc_read_runner.o
Building test test_adc_read for stm32f0xx
T: test_gpio_interrupts.c -> test_gpio_interrupts.o
Generating test_gpio_interrupts_runner.c
T: test_gpio_interrupts_runner.c -> test_gpio_interrupts_runner.o
Building test test_gpio_interrupts for stm32f0xx
make: *** No rule to make target 'ms-helpers', needed by 'solar'.  Stop.
Error: Process completed with exit code 2.
```

Reply:
oh shoot whoops